utils/utils.go

```diff
@@ -258,18 +271,30 @@ func PowerOFF() {
 	if IsOpenRCBased() {
 		SH("poweroff") //nolint:errcheck
 	} else {
-		SH("shutdown") //nolint:errcheck
+		SH("shutdown now") //nolint:errcheck
```

Comment:
I think the change is good, specially since we already have a 5 second delay here:

https://github.com/kairos-io/kairos-agent/blob/347a30b261d055b4876dd298816a9d503cdbb926/internal/agent/hooks/lifecycle.go#L20-L21

But this changes current functionality, so just bringing it up for a double check. Any reason why we need to wait the default 1min that we had before? @kairos-io/maintainers

Reply:
No idea, I assumed we just put the shortest/default command to shutdown. I could be wrong.

Reply:
Let's not do this change here and instead do it independently of the k0s addition https://github.com/kairos-io/kairos/issues/3126